inc/namespace.php

```diff
@@ -322,3 +331,28 @@ function disable_install_capability( array $caps, string $cap ) : array {
 	$caps[] = 'do_not_allow';
 	return $caps;
 }
+
+/**
+ * CloudFront can pass headers to the origin that provide information
+ * useful for modifying responses or performing redirects and other
+ * logic.
+ *
+ * This function reflects those headers back in responses for
+ * client side usage.
+ */
+function reflect_cloudfront_headers() {
+	$headers = [
+		'CloudFront-Is-Desktop-Viewer',
+		'CloudFront-Is-Mobile-Viewer',
+		'CloudFront-Is-SmartTV-Viewer',
+		'CloudFront-Is-Tablet-Viewer',
+		'CloudFront-Viewer-Country',
```

Comment:
Apart from the Viewer Country, I'm wondering if we should add these. I don't think we'd ever configure cloudfront to segment caches based off `CloudFront-Is-Tablet-Viewer`, as we'd expect that detection to be done on the client. Are there are of these apart from country that can not be done on the client-side?

Reply:
So this only reflects the headers if they're set. I'm not suggesting we enable any of the others besides `CloudFront-Viewer-Country` by default on our stacks.